src/ClassMethodsHydrator.php

```diff
@@ -194,6 +179,27 @@ public function extract(object $object) : array
         return $values;
     }
 
+    private function initCompositeFilter($object) : Filter\FilterComposite
+    {
+        if ($object instanceof Filter\FilterProviderInterface) {
+            return new Filter\FilterComposite(
+                [$object->getFilter()],
+                [new Filter\MethodMatchFilter('getFilter')]
+            );
+        }
+
+        return $this->getCompositeFilter();
+    }
+
+    private function identifyAttributeName($object, $method)
```

Comment:
we can add typehints: `object $object, string $method` and RTH: `: string`

Reply:
I need to have this habit. I applied it